Tests/XCUITests/HistoryTests.swift

```diff
@@ -103,215 +113,264 @@ class HistoryTests: BaseTestCase {
         XCTAssertEqual(app.alerts.buttons["OK"].exists, false)
     }
 
-    func testRecentlyClosedOptionAvailable() {
```

Comment:
This test has been broken into `testRecentlyClosedNoWebsiteOpen`, `testRecentlyClosedWebsiteOpen`, `testRecentlyClosedWebsiteClosed` and `testRecentlyClosedPrivateMode`.

Reply:
When a new test is created it is enabled in the test plan/scheme that is selected. As you can see in these pictures, those tests are enabled in both Smoketest3 and FullFunctional, since the test belongs to the latter, these tests should be enabled only there. Could you please disabled them from the Smoketest3 test plan?
<img width="342" alt="Screenshot 2022-10-26 at 10 14 49" src="https://user-images.githubusercontent.com/1897507/197981326-daefaab0-5e30-4a76-a363-c480ae980cb5.png">

<img width="347" alt="Screenshot 2022-10-26 at 10 14 39" src="https://user-images.githubusercontent.com/1897507/197981502-b5058d5b-64c2-4e8f-94d6-48c1390ec738.png">